packages/renderer/src/lib/preferences/PreferencesCliToolsRendering.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+import { cliToolInfos } from '../../stores/cli-tools';
+import Markdown from '../markdown/Markdown.svelte';
+import SettingsPage from './SettingsPage.svelte';
+import EngineIcon from '../ui/EngineIcon.svelte';
+import EmptyScreen from '../ui/EmptyScreen.svelte';
+</script>
+
+<SettingsPage title="CLI Tools">
+  <div class="h-full">
+    <EmptyScreen
+      aria-label="no-resource-panel"
+      icon="{EngineIcon}"
+      title="No CLI tool has been registered"
+      message="Start an extension that registers a CLI"
+      hidden="{$cliToolInfos.length > 0}" />
+
+    {#each $cliToolInfos as cliTool}
+      <div class="bg-charcoal-600 mb-5 rounded-md p-3 divide-x divide-gray-900 flex">
```

Comment:
I feel like the below section can be put into a different modal?

it'll also be easier when writing the tests for the .svelte